### PR TITLE
JENKINS: Add support for ROCm tests to Jenkins

### PR DESCRIPTION
**Pull request by souravzzz**
## Why ?
The goal is to have ROCm tests in CI and ensure any breakages are caught early

## How ?
A new module for ROCm `dev/rocm` has been added ~needs to be added called `dev/rocm-3.0`~

https://gist.github.com/souravzzz/dd04a1e4f2a354f7657af37e317cbb5a

### CODE REVIEW

**Comment by shamisp** [comment]
ok to test

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 22 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/70f4235097c860370a1ff3d77eee4eb22726fc53/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/cff0bb13b23fdce46e16e58e20eec310e77752bd/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/1b52d09a2b06f9929f2131a481a49b0a133de16e/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/d9524d45b8e8c43aa2eda9e5f43215f5bd5337d2/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/986f86eeefc37a208a5669009a60db0c86f0652d/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/3f9e51c123724f7037736e5f3e60c0badd25747d/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/69788bd9859fc08efd3f39d969d18c05939da467/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/da5d0e7644163a2d0a3dabd33c509e37ebc31c48/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/19e3ee875fbd85d75225766a545a3911d9fa4869/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/c2b72daa1bfebed49a03da5e455a789d5f4bf02e/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/22e91a05741770d1b16488f8d942612f7a25be9d/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/2ecc1618d1c99ad000ab42c716f604b1e93e3750/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/d9bbb567685b060eb03fda7f68b6958e10a82a7c/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/d005bfa2476d2f1c6791637f1b98be5ad0375a50/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/1c87feb03be212aa936f7c96b6dcedc5cad15569/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/a45348c3033aa587800dcc4a7d3d657389e8c54a/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/5a4de330a7ad0befdc955da91552ff8ab5b4acfc/hpc-test-node-new_W1) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/f5bccb9a66c4d08f8c1e8c99b27b8b86c0ec9d74/hpc-test-node-new_W2) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/285e56a0f6d5ed635e174b7498a3f6860b651e8c/hpc-test-node-new_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/539adc46084f0e528f923c747674c07fca73221b/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/8cd11c21af68496602df7a5a017c4a1b1c39b150/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/c3cde811cef934f36fc78abb46d44825a1e51096/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/07290491ae5574235fc529834255a37ceb8915f4/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/b69df7195a05c91eab0ffaf84b78c9e7511b8c80/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/c6ab650f1c6d82fa554b18f61f81666e/raw/eae6bd1376433bf3851845efc8af72d2c0dd8335/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 22 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/4dc1280c783fc7afef54c49133ada3b06f3eb719/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/6ae8818272f3ef14f959c26bd9bc49b8db87751b/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/b0f7347ebda60ba5a9ae9c5da7be58225fd2e5d3/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/18aaa980db7d0bf71073d0465720f75c7bc5b965/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/993548700607454a7f8ed17f4db962846769697e/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/9777e0826297ef92f034b621ac763ccd94ab1510/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/2f92d1a396f816f5bc6ca7611d4d1f8f302f9a86/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/f5dcd3c3cd17ed779ca6ef9cfd3fa6806ef955a5/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/9c2c7d39347694bd2cd9ee8ea940a08671466605/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/3dd6e506fc73f99160f570114bce73500fdf7395/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/aa2350f58a05ba01118e165b348e73e6a4becf76/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/c89a28832815492df2c9490817bdafd478544905/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/5b3eeb5987a3bb3e201d1eb961b83fd1d8db6244/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/d489bbe0bfe1a462e3d9274e4383a06f7ef2c9cd/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/1af6ab9f0e107bee5397098a355183065eaf74df/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/0b2e7440d238bdcddc1a9236a28f1db3ca8dcd0d/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/63f250a572f6669ce980ddba7cd7166ba7d0f2bb/hpc-test-node-new_W1) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/fe3c7cbd166ec5a4afdf5eeea5abf80bd84581c0/hpc-test-node-new_W2) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/246aee13f030b483c05daeb6168e3b6085134e20/hpc-test-node-new_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/2bb629e5f3232f808c43e863379e9cc8810fcf86/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/95cb61ce53627d6b41b78f141aca86720b998bf3/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/50e982ec1aefbf016b4d884d5175a2aa516ebcd2/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/4de0659eda12f3f7ba43e74183baefc7be460716/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/3674e1b959b7e331102e2df74f51af21554f8b66/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/343f53a1d2f5c8c90dfbccfb0957c45d/raw/0688c6abc45c76937614eb0dd09a0f1b77f02aa9/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/60674bb517cb3590fbe9afe717bfb21e42a87fd1/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/cfc7fc9c69a36ed8d5cc04f192910e325cb0e971/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/0223a9c23aa852c1b51e6fcb003d82f51a4890bb/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/4d866d58e89a1f0382e43c315340c584bfb87231/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/4f2cbbdc84c3693953a9907d08c859f78e9c4286/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/d5eca1213b97ecbae3076d7255f5180d7b1f50a6/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/b5b60c06a21c5a5de0fa3d7955bc6a805094f63b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/e13fc6523245c30179bc7e31130132be82a133a6/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/06806462c10e2fdcf8ba39d1d43ad5fffd21b193/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/362429d74cda79dea535060ddfc0d17489d3fa09/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/cdc4c2918023bcc116f4a4a3cdebd620fdcfe08f/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/5ca5481ecf0edbed915c7d562f499f58970eb4dc/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/99bae936f3130bb7ab09b89cb2347b8f6e017583/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/4dd247d683635814e2be2ee706b2aedae6230867/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/a870fa9cae9ba14d8954565a937dfe14a2c3cb04/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/862fbdaa3b891ab5392d60fafa3b45b7178fc1a7/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/c5aedddc2d3dc133950b7b9e585916310b6129b9/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/91a293a509e354ae6162c30a4023d114d7a9800e/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/947f04cbad3e4823b4a61752b4a6dcbeb4e9c4c8/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/561fbf25e91412b9783188d247f1e5c0b58ea13b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/b81fdc52c3edf2c8dea8dc076769c41f4e8da7e8/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/f6d93f8e2168bed05ffb547e02414581974cdc31/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/23806cecfb4814d8aae5d3dd171ad5e34738ba95/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/03757bcc5d309247d2a61efaa38d261b0ca14cf5/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/670cf96478b9e39550c5e7d9dae2e2f1/raw/b1cdf8a19c3b29a760fd180f0d9ef3b8f07cb4bf/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -979,6 +1024,86 @@ run_ucx_perftest() {
 
 		unset CUDA_VISIBLE_DEVICES
 	fi
+
+	# run rocm tests if rocm module was loaded and GPU is found
```
Comment (dmitrygx) [inline contrib/test_jenkins.sh]: can we introduce a function that accepts all needed arguments (e.g. kernel module ("amdgpu", "nv_peer_mem"), env variable name ("HIP_VISIBLE_DEVICES", "CUDA_VISIBLE_DEVICES") and etc) and run CUDA/ROCm perftests?

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 20-Jan-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/e3c3d508465f487277b61cff24c06468d97343bc/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/3914c2f08778d0ad586a86c8544962cbebc43515/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/38c7674263d8837a40e4d760db078aa0ca9ceafe/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/099ba331aa49d3f3088fef347831655f6ad05963/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/44345e9e137311517ca910721a6e891623b8a138/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/e64415443ff5a8fb2abd1249741c3b8e0accccc5/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/7f8ec7be92236649c607a88f108a4ac1f3db27da/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/bc3ccd134dc349b95a3d38141541116739325485/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/12f5bcb3e8815770b190d426c61557f09fa745ba/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/617fa00598fa31c5c6eed123c6b37f5715f1b312/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/6fafd2bbe67b4db7700a718e596b8beb90c1126e/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/902beaa05c78ef94def39453f67128856b903f71/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/50ed8e0fa5b7fb959fd609c340349ed0614ba987/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/ddc00899bab8fe76fa6abacfa707a355ddb68d36/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/3cace9bf95b311f412ac08dac0476cc222f50e9f/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/65131409875003018a5944d044fb10d1aa10f591/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/13a99d1ea3280f52da30157db5711fbf7ba23f50/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/590a166596480c66eb3a5d3b60c0d460aa1d66bf/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/4959c505848233daf2c5d0038633804345c16bc5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/0e9029741014b8ec653c12fe044227afed40198b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/67cbc9cc769df941baf455bd40af7198cbfba6e9/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/4787998e887e8a54a9aeedaba75f1a4a4b46248f/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/c54bbd5a195972ba04deda44e0d9c06a70f26c11/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/f0979e04365a5d0bfbc8d1d2f0f24505ca0f4ebc/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/49a53555262c7966fb010d26d864ce94/raw/a7e308d5518538419b4eb50e5198006d16e5f098/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: contrib/test_jenkins.sh (hunk 1 above)
Comment (souravzzz) [reply contrib/test_jenkins.sh]: Can be a bit tricky due the number of different environment variables. Wanted to fix the errors first before trying to refactor.

File: contrib/test_jenkins.sh [hunk 2]
```diff
@@ -118,6 +119,24 @@ unload_cuda_env() {
...
 }
 
+#
+# try load rocm modules if amd driver is installed
+#
+try_load_rocm_env() {
+	num_gpus=0
+	have_rocm=no
+	have_gdrcopy=no
+    if (lsmod | grep -q amdgpu) ; then
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: minor: indentation

File: contrib/test_jenkins.sh (hunk 1 above)
Comment (yosefe) [reply contrib/test_jenkins.sh]: agree with @dmitrygx , this is too much ~code~script duplication..

**Pull request closed without merging**